src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -603,7 +603,10 @@ private void InitializeTrainingGraph(IDataView input)
                     (string)labelType.ToString());
             }
 
-            _classCount = labelCount == 1 ? 2 : (int)labelCount;
+            var msg = $"Only one class found in {_options.LabelColumnName} column. To build a multiclass classification model, the number of classes needs to be 2 or greater";
+            Contracts.CheckParam(labelCount > 1, nameof(labelCount), msg);
```

Comment:
I've implemented the message that @natke suggested.

I had to use the "var msg" to store the string, because I couldn't use string interpolation inside the CheckParam() argument, because of this:

https://github.com/dotnet/machinelearning/blob/fa75f4fa93a252d15f05893ddbd1a22b9983cbf4/tools-local/Microsoft.ML.InternalCodeAnalyzer/Descriptions.Designer.cs#L64-L71

So the suggestion there is to use Contracts.Except() instead of Contracts.Param() if I want to use string interpolation. Problem is Contracts.Except() throws an `InvalidOperationException` (which @justinormont and @harishsk [had suggested](https://github.com/dotnet/machinelearning/pull/4662#discussion_r367778352) I shouldn't use). So I guess it's best to simply create the "msg" string before passing it to the Contracts.CheckParam()...